consensus/validator.go

```diff
@@ -173,11 +173,13 @@ func (consensus *Consensus) onPrepared(recvMsg *FBFTMessage) {
 		consensus.getLogger().Debug().Msg("[onPrepared] consensus received message before init. Ignoring")
 		return
 	}
-	if err := consensus.BlockVerifier(&blockObj); err != nil {
-		consensus.getLogger().Error().Err(err).Msg("[OnPrepared] Block verification failed")
-		return
+	if !consensus.FBFTLog.IsBlockVerified(&blockObj) {
```

Comment:
Since this piece of code is used many times, can we extract it to a function?

Reply:
done.